lib/Onyx.ts

```diff
@@ -210,8 +210,18 @@ function disconnect(connectionID: number, keyToRemoveFromEvictionBlocklist?: Ony
  * @param value value to store
  */
 function set<TKey extends OnyxKey>(key: TKey, value: NonUndefined<OnyxEntry<KeyValueMapping[TKey]>>): Promise<void> {
-    // check if the value is compatible with the existing value in the storage
-    const existingValue = cache.getValue(key, false);
+    if (OnyxUtils.hasPendingMergeForKey(key)) {
+        delete OnyxUtils.getMergeQueue()[key];
+    }
```

Comment:
Yes, because `Onyx.set` was triggered after all prior `Onyx.merge` calls, we will get rid of the merge queue and just set the value.

This isn't new, i just moved this code block to the top of the file

Reply:
Yep, that makes sense, just wanted to get this confirmed. Thank you!